README.md

```diff
@@ -111,6 +111,7 @@ Then build it like so:
 % cmake --build llvm-build
 % cmake --install llvm-build --prefix llvm-install
 ```
+Running a serial build will be slow. To improve speed, try running a parallel build. That's done by default in Ninja; for make, use the option -j NNN, where NNN is the number of parallel jobs, e.g. the number of CPUs you have. 
```

Comment:
Could you please wrap this to 80 columns, like the rest of the file?

Reply:
@alexreinking  
Done, 
sorry for the inconvenience